internal/jac/jac.go

```diff
@@ -14,10 +14,21 @@ import (
 	"strings"
 )
 
+var dependency = &dependencies.Dependency{
+	Command:    "jac",
+	Url:        "https://github.com/nestoca/jac",
+	IsRequired: false,
+	RequiredBy: []string{"project owners", "release owners"},
```

Comment:
Something important here is that I really, really, think we should install jac and clone the people repository for the user. Currently, it adds a fair amount of friction for the user to get going.